src/service.rs

```diff
@@ -12,10 +12,10 @@ use winapi::um::{winnt, winsvc};
 
 use crate::double_nul_terminated;
 use crate::sc_handle::ScHandle;
-use crate::{ErrorKind, Result, ResultExt};
+use crate::ErrorKind;
```

Comment:
Nitpick. But could you not just import `crate::Result` and have that handy type definition available? So you don't need to change all the return type declarations below.

Reply:
done!